test/unit/domain/participants.test.ts

```diff
@@ -0,0 +1,71 @@
+import { consentDB } from '~/lib/db'
+
+import {
+  retrieveConsent,
+  createConsent,
+  updateConsent,
+  deleteConsent }
+from '~/domain/participants'
+
+import { logger } from '~/shared/logger'
+import { Consent } from '~/model/consent';
+
+jest.mock('~/shared/logger')
+
+// Declare Mocks
+const mockInsertConsent = jest.spyOn(consentDB, 'insert')
+const mockUpdateConsent = jest.spyOn(consentDB, 'update')
+const mockRetrieveConsent = jest.spyOn(consentDB, 'retrieve')
+const mockDeleteConsent = jest.spyOn(consentDB, 'delete')
+const mockConsent : Consent = {
+  id: 'de51442d-f443-49c8-8021-f563daa3206a',
+  fspId: 'dfspa'
+}
+
+describe('server/domain/consents', (): void => {
+  beforeAll((): void => {
+    mockInsertConsent.mockResolvedValue(true)
+    mockUpdateConsent.mockResolvedValue(1)
+    mockDeleteConsent.mockResolvedValue(1)
+    mockRetrieveConsent.mockResolvedValue(mockConsent)
+  })
+
+  beforeEach((): void => {
+    jest.clearAllMocks()
```

Comment:
maybe it is worth to setup jest.config to automatically clear all mocks

Reply:
done.